### PR TITLE
DOC-5397: Added gateway processor metrics

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+table th:nth-of-type(4) {
+    width: 45%;
+}
+</style>
+
+| Metric | Type | Tags | Description |
+| :----------- | :----------- | :----------- | :----------- |
+| `corda_p2p_gateway_inbound_request_time_seconds` | Timer | <ul><li>`response.type`</li><li>`endpoint.source`</li></ul> | The latency and the number of requests from a gateway worker to peer gateway workers in other clusters. |
+| `corda_p2p_gateway_outbound_request_time_seconds` | Timer | <ul><li>`response.type`</li><li>`endpoint.destination`</li></ul> | The number of incoming requests from peer gateway workers in other clusters and the time it took to process them. Main way of processing on the inbound requests is identifying the partition where a session is hosted and writing the message to Kafka, so that it’s later processed by the appropriate link manager. |
```
Comment (dimosr) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md:50]: `Main way of processing` doesn't sound right. In the original doc, it was `main path of processing` which refers to what's executed as part of the processing.

But, I think we can just remove this sentence altogether as it's quite low-level detail for an external audience that's not present in other metrics.

**Review by dimosr — COMMENTED** [review]
just a minor comment

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md]: Removed the sentence.

**Review by dimosr — APPROVED** [review]

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    identifier: corda5-cluster-gateway-worker
+    weight: 13000
+section_menu: corda5
+---
+
+# Gateway Worker
+
+The P2P {{< tooltip >}}gateway worker{{< /tooltip >}} is tasked with sending outbound messages to virtual nodes located in
+different clusters and receiving inbound messages from virtual nodes within different clusters. To accomplish this,
+a gateway worker establishes TLS connections with gateway workers in other clusters and exchanges messages through HTTPS requests.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md:17]:
```suggestion
a gateway worker establishes {{< tooltip >}}TLS{{< /tooltip >}} connections with gateway workers in other clusters and exchanges messages through HTTPS requests.
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+In the outbound direction, internal components within the cluster forward messages intended for the gateway worker through a
+message bus (for example, Kafka). Similarly, in the inbound direction, the gateway worker forwards any messages received via HTTPS
+from other clusters to the local cluster through the message bus.
+
+The P2P gateway worker is additionally tasked with conducting certificate revocation checks. These checks are carried out
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md:23]:
```suggestion
The P2P gateway worker is additionally tasked with conducting certificate {{< tooltip >}}revocation checks{{< /tooltip >}}. These checks are carried out
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+| `corda_p2p_gateway_outbound_request_time_seconds` | Timer | <ul><li>`response.type`</li><li>`endpoint.destination`</li></ul> | The number of incoming requests from peer gateway workers in other clusters and the time it took to process them. |
+| `corda_p2p_gateway_inbound_tls_connections_count` | Counter | <ul><li>`connection.result`</li><li>`endpoint.source`</li></ul> | The number of inbound TLS connections from other gateway workers. Connections are kept open while there is activity. |
+| `corda_p2p_gateway_outbound_tls_connections_count` | Counter | <ul><li>`connection.result`</li><li>`endpoint.destination`</li></ul> | The number of outbound TLS connections from other gateway workers. Connections are kept open while there is activity. |
+| `corda_p2p_gateway_cert_revocation_check_time_seconds` | Timer | None | The number of certificate revocation check requests and the time it took to process them. These requests are sent to the gateway worker from internal components when they want to check revocation of a certificate. An example is the link manager wanting to check revocation of session certificates when these are used (by default, session PKI is turned off). |
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics/gateway-worker.md:52]:
```suggestion
| `corda_p2p_gateway_cert_revocation_check_time_seconds` | Timer | None | The number of certificate revocation check requests and the time it took to process them. These requests are sent to the gateway worker from internal components when they want to check revocation of a certificate. An example is the link manager checking revocation of session certificates when these are used (by default, session PKI is turned off). |
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — COMMENTED** [review]
LGTM but just need the correct folder.

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
Thx @AnnaK-R3

**Pull request merged**